node_test/paymentproxy_test.go

```diff
@@ -185,7 +185,7 @@ func createVoucher(t *testing.T, client rpc.RpcClientApi, channelId types.Destin
 }
 
 func expectedPaymentErrorMessage(total, numPaid int) string {
-	return fmt.Sprintf("payment of %d required, the voucher only resulted in a payment of %d", total, numPaid)
+	return fmt.Sprintf("payment of %d attoFIL required, the voucher only resulted in a payment of %d attoFIL", total, numPaid)
```

Comment:
Here is some Filecoin-specific code which doesn't belong here. We can fix that as a part of https://github.com/statechannels/go-nitro/issues/1785.

Reply:
https://github.com/statechannels/go-nitro/issues/1816